libphal/phal_exception.H

```diff
@@ -120,27 +124,32 @@ class SbeError final : public phalError
 	{
 		if (!ffdcFileList.empty()) {
 			const auto &it = ffdcFileList.begin();
-			return it->second.first;
+			const auto& tuple = it->second;
+			return std::get<1>(tuple);
```

Comment:
Rather than doing a `const auto &it = ffdcFileList.begin();` and `const auto& tuple = it->second;` what we can do for more clarity is something like below
`auto errSeverity =  std::get<1>(ffdcFileList.cbegin()->second)` or we can comment next to LN128 like
`return std::get<1>(tuple); //Returning error severity from FFDC file list`. 
This way the reader would come to know what we are exactly returning to the caller. Just a suggestion.